src/main.ts

```diff
@@ -406,15 +420,16 @@ export default class RecipeGrabber extends Plugin {
 		filename: Recipe["name"],
 		imgUrl: Recipe["image"],
 		file: TFile,
-		imgNum = false,
```

Comment:
eslint wants us to keep value as number, here it was being used as both number and bool